### PR TITLE
ScanTaskRunners Refactorings

### PR DESCRIPTION
**Pull request by bruno-fs**
These refactorings will contribute to DISCOVERY-186 implementation

### CODE REVIEW

**Comment by nicolearagao** [comment]
> These refactorings will contribute to DISCOVERY-186 implementation

@bruno-fs I guess the right jira ticket is DISCOVERY-187

File: quipucords/tests/factories.py [hunk 1]
```diff
@@ -123,3 +127,43 @@ class Meta:
         """Factory options."""
 
         model = "api.ScanJob"
+
+
+class CredentialFactory(DjangoModelFactory):
```
Comment (nicolearagao) [inline quipucords/tests/factories.py:132]: ❤️ 🤘🏼

**Review by nicolearagao — APPROVED** [review]
great work! 👍🏼

**Review by gtanzillo — APPROVED** [review]
Looks great!! 👍

**Pull request merged**